tests/sentry/grouping/test_enhancer.py

```diff
@@ -30,6 +32,7 @@ def dump_obj(obj):
 
 @pytest.mark.parametrize("version", [1, 2])
 @django_db_all
+@override_options({"grouping.rust_enhancers.parse_rate": 1.0})
```

Comment:
For this whole test file, should we set both of them to 1.0 so all tests are tested?

You may want to run the tests with `-s -vv` to see if any exceptions are being hidden (which is fine but it may save us reviewing non-blocking errors later in production).

Reply:
Most of these tests run the python-specific `get_matching_frame_actions` which does not exist directly in Rust.
I made sure to add all these sampling rates to `tests/sentry/grouping/test_categorization.py` which runs the enhancement rules on a huge chunk of fixtures.